src/helpers/utils.ts

```diff
@@ -19,9 +19,10 @@ export async function getSubscribers(space) {
       __args: {
         where: { space }
       },
-      address: true
+      address: true,
```

Comment:
We shouldn't add extra comma here
```suggestion
      address: true
```